.github/workflows/ci-build.yml

```diff
@@ -11,34 +11,49 @@ on:
 
 jobs:
   test:
-    name: Tests
+    name: "Tests"
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v2
+      - name: Install Node
+        uses: actions/setup-node@v2
         with:
           node-version: 12.x
-      - name: install dependencies
-        run: yarn install
-      - name: lint
+          cache: yarn
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+      - name: Lint
         run: |
           yarn lint:hbs
           yarn lint:js
-      - name: test
+      - name: Run Tests
         run: yarn test
 
-  try-scenarios:
-    name: ${{ matrix.ember-try-scenario }}
-
+  floating:
+    name: "Floating Dependencies"
     runs-on: ubuntu-latest
 
-    needs: test
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
+          cache: yarn
+      - name: Install Dependencies
+        run: yarn install --no-lockfile
+      - name: Run Tests
+        run: yarn test
+
+  try-scenarios:
+    name: ${{ matrix.try-scenario }}
+    runs-on: ubuntu-latest
+    needs: 'test'
 
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
Why do we not want to fail fast?

Reply:
This aligns with latest addon blueprint and more details in https://github.com/ember-cli/ember-cli/pull/9661

if you disagree - I can revert

Reply:
Cool. I'm fine with aligning with the default!